common/persistence/class.SqlKvDriver.php

```diff
@@ -181,24 +184,42 @@ public function del($id)
     /**
      * Increment existing value
      * @param string $id
-     * @return mixed
+     * @return boolean
      */
     public function incr($id)
     {
+        switch ($this->sqlPersistence->getPlatForm()->getName()) {
+            case 'psql':
+                $statement = 'UPDATE kv_store SET kv_value = kv_value::integer + 1 WHERE kv_id = :id';
+                break;
+            case 'gcp-spanner':
+                $statement = 'UPDATE kv_store SET kv_value = CAST(CAST(kv_value as INT64) + 1 as string) WHERE kv_id = :id';
+                break;
+            default:
+                $statement = 'UPDATE kv_store SET kv_value = kv_value + 1 WHERE kv_id = :id';
+        }
         $params = [':id' => $id];
-        $statement = 'UPDATE kv_store SET kv_value = kv_value + 1 WHERE kv_id = :id';
         return $this->sqlPersistence->exec($statement, $params);
     }
 
     /**
      * Decrement existing value
      * @param $id
-     * @return mixed
+     * @return boolean
      */
     public function decr($id)
     {
+        switch ($this->sqlPersistence->getPlatForm()->getName()) {
+            case 'psql':
```

Comment:
I've gotten `'postgresql'` platform name
(and it was:         `$intVal = $platformName == 'postgresql' ? 'kv_value::integer' : 'kv_value';` in previous implementation)

Reply:
😭 blame on me